diff -rupN eigen-eigen-323c052e1731/CMakeLists.txt eigen-eigen-323c052e1731-new/CMakeLists.txt
--- eigen-eigen-323c052e1731/CMakeLists.txt	2018-12-11 18:57:55.000000000 +0100
+++ eigen-eigen-323c052e1731-new/CMakeLists.txt	2018-12-25 23:10:48.527420853 +0100
@@ -424,6 +424,11 @@ install(FILES
   DESTINATION ${INCLUDE_INSTALL_DIR} COMPONENT Devel
   )
 
+install(FILES
+  cmake/FindEigen3.cmake
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/share/cmake/Modules
+  )
+
 if(EIGEN_BUILD_PKGCONFIG)
     configure_file(eigen3.pc.in eigen3.pc @ONLY)
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/eigen3.pc
