diff -rupN eigen-eigen-323c052e1731/eigen3.pc.in eigen-eigen-323c052e1731-new/eigen3.pc.in
--- eigen-eigen-323c052e1731/eigen3.pc.in	2018-12-11 18:57:55.000000000 +0100
+++ eigen-eigen-323c052e1731-new/eigen3.pc.in	2018-12-25 23:10:48.594420575 +0100
@@ -6,4 +6,4 @@ Description: A C++ template library for
 Requires:
 Version: @EIGEN_VERSION_NUMBER@
 Libs:
-Cflags: -I${prefix}/@INCLUDE_INSTALL_DIR@
+Cflags: -I@INCLUDE_INSTALL_DIR@
