diff -rupN eigen-eigen-36fd1ba04c12/Eigen/src/CholmodSupport/CholmodSupport.h eigen-eigen-36fd1ba04c12-new/Eigen/src/CholmodSupport/CholmodSupport.h
--- eigen-eigen-36fd1ba04c12/Eigen/src/CholmodSupport/CholmodSupport.h	2014-12-16 18:30:52.000000000 +0100
+++ eigen-eigen-36fd1ba04c12-new/Eigen/src/CholmodSupport/CholmodSupport.h	2014-12-17 23:35:28.080925651 +0100
@@ -78,7 +78,7 @@ cholmod_sparse viewAsCholmod(SparseMatri
   {
     res.itype = CHOLMOD_INT;
   }
-  else if (internal::is_same<_Index,UF_long>::value)
+  else if (internal::is_same<_Index,SuiteSparse_long>::value)
   {
     res.itype = CHOLMOD_LONG;
   }
diff -rupN eigen-eigen-36fd1ba04c12/Eigen/src/SPQRSupport/SuiteSparseQRSupport.h eigen-eigen-36fd1ba04c12-new/Eigen/src/SPQRSupport/SuiteSparseQRSupport.h
--- eigen-eigen-36fd1ba04c12/Eigen/src/SPQRSupport/SuiteSparseQRSupport.h	2014-12-16 18:30:52.000000000 +0100
+++ eigen-eigen-36fd1ba04c12-new/Eigen/src/SPQRSupport/SuiteSparseQRSupport.h	2014-12-17 23:35:28.063925342 +0100
@@ -47,7 +47,7 @@ namespace Eigen {
  * You can then apply it to a vector.
  * 
  * R is the sparse triangular factor. Use matrixQR() to get it as SparseMatrix.
- * NOTE : The Index type of R is always UF_long. You can get it with SPQR::Index
+ * NOTE : The Index type of R is always SuiteSparse_long. You can get it with SPQR::Index
  * 
  * \tparam _MatrixType The type of the sparse matrix A, must be a column-major SparseMatrix<>
  * NOTE 
@@ -59,7 +59,7 @@ class SPQR
   public:
     typedef typename _MatrixType::Scalar Scalar;
     typedef typename _MatrixType::RealScalar RealScalar;
-    typedef UF_long Index ; 
+    typedef SuiteSparse_long Index ; 
     typedef SparseMatrix<Scalar, ColMajor, Index> MatrixType;
     typedef PermutationMatrix<Dynamic, Dynamic> PermutationType;
   public:
